### PR TITLE
doc: ActionCable overview grammar and wording fixes [ci-skip]

### PR DESCRIPTION
**Pull request by mihaic195**
Minor ActionCable overview grammar and wording fixes.

### CODE REVIEW

File: guides/source/action_cable_overview.md [hunk 1]
```diff
@@ -399,8 +399,8 @@ consumer.subscriptions.create({ channel: "ChatChannel", room: "Best Room" }, {
 
 ### Passing Parameters to Channels
 
-You can pass parameters from the client-side to the server-side when creating a
-subscription. For example:
+When creating a subscription, you can pass parameters from the client-side to the
+server-side. For example:
```
Comment (jonathanhefner) [inline guides/source/action_cable_overview.md:403]: Let's keep this as-is:

```suggestion
You can pass parameters from the client-side to the server-side when creating a
subscription. For example:
```

File: guides/source/action_cable_overview.md [hunk 2]
```diff
@@ -626,11 +626,11 @@ key being passed to `current_user.appear`.
 
 The appearance example was all about exposing server functionality to
 client-side invocation over the WebSocket connection. But the great thing
-about WebSockets is that it's a two-way street. So now let's show an example
+about WebSockets is that it's a two-way street. So now, let's show an example
```
Comment (jonathanhefner) [inline guides/source/action_cable_overview.md:629]: I think this reads fine without the comma, but, if we do add that comma, we should add another:

```suggestion
about WebSockets is that it's a two-way street. So, now, let's show an example
```

File: guides/source/action_cable_overview.md [hunk 3]
```diff
@@ -881,14 +881,14 @@ The Ruby side of things is built on top of [websocket-driver](https://github.com
 
 ## Deployment
 
-Action Cable is powered by a combination of WebSockets and threads. Both the
+A combination of WebSockets and threads powers Action Cable. The
```
Comment (jonathanhefner) [inline guides/source/action_cable_overview.md:884]: Let's keep this as-is:

```suggestion
Action Cable is powered by a combination of WebSockets and threads. Both the
```

File: guides/source/action_cable_overview.md [hunk 4]
```diff
@@ -881,14 +881,14 @@ The Ruby side of things is built on top of [websocket-driver](https://github.com
...
 framework plumbing and user-specified channel work are handled internally by
-utilizing Ruby's native thread support. This means you can use all your regular
+utilizing Ruby's native thread support. This means you can use all your typical
```
Comment (jonathanhefner) [inline guides/source/action_cable_overview.md:886]:
```suggestion
utilizing Ruby's native thread support. This means you can use all your existing
```

**Review by p8 — CHANGES_REQUESTED** [review]
Hi @mihaic195 ! Thanks for contributing to Rails.
We try to avoid cosmetic changes (only commas, synonyms, etc).
Please see https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#write-your-code for more details.

File: guides/source/action_cable_overview.md [hunk 5]
```diff
@@ -43,15 +43,15 @@ WebSockets open to your application if they use multiple browser tabs or devices
 ### Consumers
 
 The client of a WebSocket connection is called the *consumer*. In Action Cable
```
Comment (jonathanhefner) [inline guides/source/action_cable_overview.md:45]: We can add a comma here:

```suggestion
The client of a WebSocket connection is called the *consumer*. In Action Cable,
```

File: guides/source/action_cable_overview.md [hunk 6]
```diff
@@ -881,14 +881,14 @@ The Ruby side of things is built on top of [websocket-driver](https://github.com
...
+A combination of WebSockets and threads powers Action Cable. The
 framework plumbing and user-specified channel work are handled internally by
-utilizing Ruby's native thread support. This means you can use all your regular
+utilizing Ruby's native thread support. This means you can use all your typical
 Rails models with no problem, as long as you haven't committed any thread-safety sins.
 
 The Action Cable server implements the Rack socket hijacking API,
 thereby allowing the use of a multithreaded pattern for managing connections
-internally, irrespective of whether the application server is multi-threaded or not.
+internally, irrespective of whether the application server is multithreaded or not.
```
[the remark this answers is not in the document]
Comment (mihaic195) [reply guides/source/action_cable_overview.md]: I saw it on line 890 and I thought it was the right way 😅

**Comment by mihaic195** [comment]
> Hi @mihaic195 ! Thanks for contributing to Rails. We try to avoid cosmetic changes (only commas, synonyms, etc). Please see https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#write-your-code for more details.

Got it! Thank you!

**Comment by jonathanhefner** [comment]
@mihaic195 Looks good!  Would you mind [squashing the commits](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits)?  Then I'll give this a merge!

**Comment by mihaic195** [comment]
> @mihaic195 Looks good! Would you mind [squashing the commits](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits)? Then I'll give this a merge!

Done! Thank you!

**Comment by jonathanhefner** [comment]
Thank you, @mihaic195! :tada:

**Pull request merged**